src/main/scala/com/softwaremill/mqperf/config/TestConfig.scala

```diff
@@ -16,6 +16,7 @@ case class TestConfig(
     receiveMsgBatchSize: Int,
     brokerHosts: List[String],
     runId: String,
+    streamsCount: Int,
```

Comment:
I wouldn't add this to the "common" config, but only read the configuration value in the mq implementation